### ISSUE TITLE
[BUG] Can't create super user (Bad Documentation)

### ISSUE THREAD

**Issue by UbioZur** [paperless-ngx/paperless-ngx#884]
### Description

The documentation to create the superuser is 'bad', at least when it comes to doing it from portainer.

https://paperless-ngx.readthedocs.io/en/latest/setup.html#install-paperless-from-docker-hub
-> Nothing about the command to run

https://github.com/paperless-ngx/paperless-ngx/blob/main/docker/compose/docker-compose.portainer.yml
```
# python3 manage.py createsuperuser
python3: can't open file 'manage.py': [Errno 2] No such files or Directory
```

The actual command should be (and it's much much simplier:
`manage createsuperuser`

That command could be set inside the documentation, and the comment in the docker-compose example file should be updated.

also valid for all the following part of the documentation:
https://paperless-ngx.readthedocs.io/en/latest/search.html?q=createsuperuser&check_keywords=yes&area=default

### Steps to reproduce

1. install the docker compose through portainer
2. try to run a docker-compose run through portainer ...
3. use the python3 command from the console inside portainer ...

### Paperless-ngx version

1.7.0

### Host OS

Debian

### Installation method

Docker

**Comment by shamoon** [comment]
Actually that command works fine for me, with portainer. Is it the `USERMAP_UID` / `USERMAP_GID` ?

![port](https://user-images.githubusercontent.com/4887959/167045008-dbb9090d-6f4d-41d2-b400-1bc159851521.PNG)

**Comment by UbioZur** [comment]
> Actually that command works fine for me, with portainer. Are you in the right directory?
> 
> ![port](https://user-images.githubusercontent.com/4887959/167045008-dbb9090d-6f4d-41d2-b400-1bc159851521.PNG)

You are in the wrong directory!
Portainer open the console to `/` not to `/usr/src/paperless/src` (at least on my system)

and `manage.py` does not exist inside `/` !

but `manage createsuperuser` works inside every directories

So instead of adding the the doc that people have to move into the src directory, it's better to just make them use the manage bin I think.

**Comment by qcasey** [comment]
> but manage createsuperuser works inside every directories

As long as that works on other deployments (other than portainer) I don't see the harm in changing. We'd have to test this of course.

**Comment by shamoon** [comment]
For me it defaults to that dir, I never set that (don’t even know how).

And, that command doesn’t work for me, which I feel like this makes sense, `manage` is not a binary, is it?! I’m def not the expert here. Does `which manage` work for you?

```
root@d7a76459del7:/usr/src/paperless/src#manage createsuperuser
bash: manage: command not found
rootedla76459de17:/usr/src/paperless/src#1
```

**Comment by stumpylog** [comment]
I assume Portainer just opens the prompt into the last image WORKDIR.  Directly execing into a running container [as is consistent with the docs](https://docs.docker.com/engine/reference/commandline/exec/), so Portainer is just wrapping the exec.

There's no `manage` for me either.  It's `python3 manage.py ...`

**Comment by UbioZur** [comment]
I found the problem, and it's on my side Obviously!

I had to switch to the linuxserver image because of a redis error. 
I tried again with the official and there is in fact no `manage` and the container start at the correct path. (And I still have that redis exception error)

Still the doc (https://paperless-ngx.readthedocs.io/en/latest/setup.html#install-paperless-from-docker-hub) does not talk about running `python3 manage.py createsuperuser` for when a user is using a webUI such as Portainer. It could be a good addition to the doc.
